WooCommerce/Classes/ViewRelated/Feature Highlight/TooltipPresenter.swift

```diff
@@ -117,6 +117,13 @@ final class TooltipPresenter {
         }
     }
 
+    // Silently removes the tooltip without firing the `primaryTooltipAction` callback
+    //
+    func removeTooltip() {
```

Comment:
nit: the code seems to be the same as L115-116. Maybe this new function can be called there as well?

non-blocking future suggestion: since this is now a public function, it can also be unit tested about the notification at least.

Reply:
> the code seems to be the same as L115-116. Maybe this new function can be called there as well?

Done in 0bbb98f3c5808f6aee4c83523300d015a77932a9

Reply:
> non-blocking future suggestion: since this is now a public function, it can also be unit tested about the notification at least.

Done in 00d75c96a09c2bb72a8ed7e44211b286c5711808